evals/solvers/providers/openai/whisper_solver.py

```diff
@@ -0,0 +1,73 @@
+# from evals.solvers.providers.openai.openai_solver import OpenAISolver
+import base64
+import io
+from typing import Any, Optional
+
+import openai
+
+from evals.solvers.solver import NestedSolver, Solver, SolverResult, SolverSpec
+from evals.task_state import TaskState
+
+
+class WhisperSolver(NestedSolver):
```

Comment:
nit: suggest `WhisperCascadedSolver` or `WhisperPipelineSolver` instead to make it explicit that it does ASR + LLM.